plugins/amdgpu/amdgpu_plugin_topology.c

```diff
@@ -1241,7 +1244,7 @@ static bool map_devices(struct tp_system *src_sys, struct tp_system *dest_sys, s
 				return true;
 			} else {
 				/* We could not map remaining nodes in the list. Add dest node back
-				 * to list and try to map next dest node in list to current src
+				 * to list and try to map next dest ndoe in list to current src
```

Comment:
typo

Reply:
Posting an update that has the typo fixed.